WPMedia/fusion-news-theme-blocks
# PEN-1527 Advertisement Label Display for mobile view

## Description
- update to hide advertisement label for right rail cube in mobile

## Jira Ticket
- [PEN-1527](https://arcpublishing.atlassian.net/browse/PEN-1527)

## Acceptance Criteria
- advertisement label should be hidden in mobile view for `Right rail cube` ad type

## Test Steps
- pull this branch locally, and run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..
- in `Fusion-New-Theme` repo, run `npx fusion start -f -l @wpmedia/ads-block@canary`
- once fusion started, create a new page with `right-rail` layout, and add the `Ad Block` to the `right rail` section
<img width="529" alt="Screen Shot 2020-12-09 at 2 01 23 PM" src="https://user-images.githubusercontent.com/4411687/101693942-35761a00-3a27-11eb-81c4-48e042509cd5.png">

- in the `Ad Block` customfield, select `Right Rail Cube` Type
<img width="325" alt="Screen Shot 2020-12-09 at 2 01 41 PM" src="https://user-images.githubusercontent.com/4411687/101693967-3f981880-3a27-11eb-980b-e3a3a1bea6a0.png">

- stage and published the page
- in regular desktop view, you should be able to see the `Advertisement` label
- open chrome developer tool, and select to mobile view and refresh the page
- once the page is refreshed, the advertisement label should not be shown.


## Effect Of Changes
### Before
advertisement label for right rail cube rendered on mobile view even though no ad is rendered.


### After
advertisement label for right rail cube NO LONGER render on mobile view.


## Dependencies or Side Effects


## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
